### PR TITLE
spell out the full name

### PR DESCRIPTION
**Pull request by wulinsheng123**
Spell out TSC's full name for someone who just joined astro

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,17 +24,17 @@
...
+**Goal:** Confirm proposal feasibility with Astro Maintainers and [Technical Steering Committee](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#technical-steering-committee-tsc).
 
 **Requirements:** An existing proposal (Stage 1). In addition, a proposal is more likely to be accepted if it is detailed and well thought-out, can demonstrate community interest, has at least one champion volunteer, and has buy-in/interest from Astro maintainer(s).
 
 **Location:** GitHub Issues [(see all accepted proposals).](https://github.com/withastro/roadmap/issues)
 
-**What to Expect:** A proposal reaches this stage (aka "is accepted") during a meeting with Maintainers and TSC, following our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
+**What to Expect:** A proposal reaches this stage (aka "is accepted") during a meeting with Maintainers and [Technical Steering Committee](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#technical-steering-committee-tsc), following our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
```
Comment (delucis) [inline README.md:33]:
```suggestion
**What to Expect:** A proposal reaches this stage (aka "is accepted") during a meeting with Maintainers and TSC, following our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
```

**Review by delucis — COMMENTED** [review]
Thanks @wulinsheng123! Left some suggestions to streamline this 🙌

File: README.md (hunk 1 above)
Comment (jasikpark) [reply README.md:33]: Maybe a footnote with the link could work for all of these? 🤷 if you're not reading this linearly

**Comment by wulinsheng123** [comment]
cc @delucis

**Review by delucis — APPROVED** [review]
Perfect — thanks for calling out and improving this @wulinsheng123 💜

**Pull request merged**